src/health_collector.ts

```diff
@@ -237,7 +237,8 @@ export default class HealthCollector {
             jicofoHealthy &&
             prosodyHealthy &&
             statusFileResult.readable &&
-            parsedStatsFlag // stats file parsed successfully
+            parsedStatsFlag &&        // stats file parsed successfully
+            statusFileResult.contents !== 'unhealthy'
```

Comment:
I believe we'll need to treat 'unhealthy' as 'drain' in the haproxy agent output.